fractal-puppy/Engine
# Enemy scripts debug draw

The AI scripts of the enemies now have a debug option to draw behavior such as distances.

To test:
- Open "EnemiesTest" scene and hit play. On the scene window, you should be able to see the debug for the enemies. Example of what you should see:
![ezgif-3-16a29b3ce082](https://user-images.githubusercontent.com/17391875/58724919-1c29f180-83de-11e9-9aee-666ecc192491.gif)
